x-pack/plugins/drilldowns/public/components/drilldown_hello_bar/drilldown_hello_bar.tsx

```diff
@@ -44,7 +44,9 @@ export const DrilldownHelloBar: React.FC<DrilldownHelloBarProps> = ({
           {docsLink && (
             <>
               <EuiSpacer size={'xs'} />
-              <EuiLink href={docsLink}>{txtViewDocsLinkLabel}</EuiLink>
+              <EuiLink href={docsLink} target="_blank" rel="noopener">
```

Comment:
1. afaik EuiLink with target="_blank" handles correct rels inside, so we can remove. 
2. also maybe we should add `external` prop to get nice "external" icon next to link text

Reply:
Done.